plugins/openai/src/realtime/realtime_model.ts

```diff
@@ -446,20 +522,48 @@ export class RealtimeSession extends multimodal.RealtimeSession {
         tools,
         tool_choice: toolChoice,
       },
-    });
+    };
+
+    if (this.#opts.provider === 'microsoft' && this.#opts.maxResponseOutputTokens === Infinity) {
+      // microsoft doesn't support inf for max_response_output_tokens, but accepts no args
+      sessionUpdateEvent.session.max_response_output_tokens = undefined;
+    }
+
+    this.queueMsg(sessionUpdateEvent);
   }
 
   #start(): Promise<void> {
     return new Promise(async (resolve, reject) => {
-      this.#ws = new WebSocket(
-        `${this.#opts.baseURL}?model=${encodeURIComponent(this.#opts.model)}`,
-        {
-          headers: {
-            Authorization: `Bearer ${this.#opts.apiKey}`,
-            'OpenAI-Beta': 'realtime=v1',
-          },
-        },
-      );
+      const headers: Record<string, string> = {
+        'User-Agent': 'LiveKit Agents JS',
```

Comment:
let's follow the [semantics](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent) for user-agents, and have something like 

```
Agents-JS/1.0.0 (+https://github.com/livekit/agents-js)
```
doesn't really change anything but if we add one let's at least make it good

Reply:
Feel free to add version if you'd like.